.pipelines/mysql-pipelines.yml

```diff
@@ -77,8 +77,32 @@ jobs:
       arguments: '--filter "TestCategory=MySql" --no-build --configuration $(buildConfiguration) --collect "XPlat Code coverage"'
       projects: '**/*Tests/*.csproj'
 
+  - task: CmdLine@2
+    displayName: 'Set flag to publish Verify *.received files when tests fail'
+    condition: failed()
+    inputs:
+      script: 'echo ##vso[task.setvariable variable=publishverify]Yes'
+
   - task: PublishCodeCoverageResults@1
     displayName: 'Publish code coverage'
     inputs:
       codeCoverageTool: Cobertura
       summaryFileLocation: '$(Agent.TempDirectory)/**/*cobertura.xml'
+
+  - task: CopyFiles@2
```

Comment:
nit: for consistency with other pipelines the publish Verify tasks can all be in the same sequence proceeded by PublishCodeCoverageResults@1

Reply:
Moved. Although I think we can improve the pipelines that that have common stuff using [Templates](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/templates?view=azure-devops).

I'll create a new task for that cleanup.